src/components/ec-metroline-cards/ec-metroline-cards.story.js

```diff
@@ -0,0 +1,95 @@
+import EcMetrolineCards from './ec-metroline-cards.vue';
+import EcIcon from '../ec-icon';
+import EcMetrolineCard from '../ec-metroline-card';
+
+export default {
+  title: 'Metroline Cards',
+  component: EcMetrolineCards,
+};
+
+export const basic = (args, { argTypes }) => ({
```

Comment:
I think the idea with the new stories is not to do bind the template to a specific scenario so we can reuse it if we want
Then again, here we only have one scenario so as you prefer.
(https://storybook.js.org/docs/react/writing-stories/introduction#using-args )

`const Template = (args, { argTypes }) => ({`

Then at the bottom of the file you can do 

```
export const basic = Template.bind({});
export const anotherStory = Template.bind({});

basic.args = {
  isCollapsible: true,
};

anotherStory.args = {
  isCollapsible: false,
};
```

If we change this here maybe worth changing at the metroline-card as well.